run.sh

```diff
@@ -8,13 +8,13 @@ function log {
 }
 
 log "Get Weather info"
-sudo -E python3 screen-weather-get.py
+python3 screen-weather-get.py
 
 current_hour=`date +"%H"`
 
 
 log "Get Calendar info"
-sudo -E python3 screen-calendar-get.py
+python3 screen-calendar-get.py
```

Comment:
The -E should have passed environment variables along but it's better this way, if sudo isn't needed, better to remove it. Thanks!